summon.lua

```diff
@@ -0,0 +1,193 @@
+local Minecarts =
+{
+	["minecart"] = E_ITEM_MINECART,
+	["chest_minecart"] = E_ITEM_CHEST_MINECART,
+	["furnace_minecart"] = E_ITEM_FURNACE_MINECART,
+	["hopper_minecart"] = E_ITEM_MINECART_WITH_HOPPER,
+	["tnt_minecart"] = E_ITEM_MINECART_WITH_TNT,
+
+	-- 1.10 and below
+	["MinecartChest"] = E_ITEM_CHEST_MINECART,
+	["MinecartFurnace"] = E_ITEM_FURNACE_MINECART,
+	["MinecartHopper"] = E_ITEM_MINECART_WITH_HOPPER,
+	["MinecartRideable"] = E_ITEM_MINECART,
+	["MinecartTNT"] = E_ITEM_MINECART_WITH_TNT
+}
+
+local Mobs =
```

Comment:
I'm not sure why @mathiascode used the tables originally, as it's his code. I shall change it to use StringToMobType